Misc/NEWS.d/next/Library/2018-11-03-12-38-03.bpo-35153.009pdF.rst

```diff
@@ -0,0 +1,2 @@
+Add ``headers`` to ``xmlrpc.client.ServerProxy``.
```

Comment:
You also modified Transport, no? Please mention it.

I don't understand why Transport isn't document at https://docs.python.org/dev/library/xmlrpc.client.html whereas it's used in an example. Maybe you want to write a separated change to document Transport class?

Reply:
There is a comment in the doc about not everything is documented yet.
I may try to write it later.